docs/extending.rst

```diff
@@ -116,14 +116,13 @@ These output functions take three parameters, ``data``, ``code``, and
 ``headers``
 
 ``data`` is the object you return from your resource method, code is the HTTP
-status code that it expects, and headers are any HTTP headers to set in the
+status code that it expects, and headers are any HTTP headers to define in the
```

Comment:
ok, i will revert back that change...let me know if there are any other changes before i update the pull request. thanks for the quick review